glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -53,6 +53,7 @@ data class GleanTimerId internal constructor(internal val id: Long)
 open class GleanInternalAPI internal constructor () {
     companion object {
         private const val LOG_TAG: String = "glean/Glean"
+        private const val PLATFORM_NAME: String = "Android"
```

Comment:
Instead of calling this `PLATFORM` here and in the other places, can we be a bit more specific about what this is about? This is really a `USER_AGENT_PLATFORM`, since we specifically want to use it for the user agent part.